lib/facter/yum_reboot_required.rb

```diff
@@ -0,0 +1,11 @@
+require 'English'
```

Comment:
what's the purpose of that?

Reply:
You get `$CHILD_STATUS` instead of `$?`

Reply:
Yes, rubocop complains otherwise:

```
lib/facter/yum_reboot_required.rb:6:8: C: Prefer $CHILD_STATUS from the stdlib 'English' module (don't forget to require it) over $?.
      !$?.success?
       ^^
```

If you prefer, I can disable this cop, i can switch to the more common (in my feeling) `$?`.  Just tell me.